Projects/Domains/AppDomain/Sources/DataSource/RemoteAppDataSourceImpl.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  RemoteAppDataSourceImpl.swift
+//  AppDomain
+//
+//  Created by KTH on 2024/03/04.
+//  Copyright © 2024 yongbeomkwak. All rights reserved.
+//
+
+import Foundation
+import BaseDomain
+import AppDomainInterface
+import RxSwift
+
+public final class RemoteAppDataSourceImpl: BaseRemoteDataSource<AppAPI>, RemoteAppDataSource {
+    public func fetchAppCheck() -> Single<AppCheckEntity> {
+        request(.fetchAppCheck)
+            .map(FetchAppCheckResponseDTO.self)
+            .map({$0.toDomain()})
```

Comment:
```suggestion
            .map { $0.toDomain() }
```
trailing 클로져 쓰는거 어떤가요?

Reply:
저도 그렇게 깔끔하고 좋은데, 린트가 경고를 내더라구요. 윗 줄 때문인가..?

Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)

Reply:
어라..? 왜날까요..?

Reply:
[].map{ $0 }
[].map(){ $0 }
이런거처럼 메서드명이랑 여는 중괄호랑 공백 없을떄 뜨는 warning인거같은데 혹시 확인해봐주실 수 있나용?

Reply:
저렇게 하니깐 경고가 1개 늘었네요. 

Empty Parentheses with Trailing Closure Violation: When using trailing closures, empty parentheses should be avoided after the method call. (empty_parentheses_with_trailing_closure)

Reply:
<img width="434" alt="image" src="https://github.com/wakmusic/wakmusic-iOS/assets/74440939/e59a911e-d884-445c-aac6-03a83e29e74e">
이럴때 뜨는 warning인거같은데 혹시 어떤상태인가요?

Reply:
<img width="634" alt="스크린샷 2024-03-06 오후 5 27 59" src="https://github.com/wakmusic/wakmusic-iOS/assets/37323252/6a4f3255-921a-422a-8e1d-3f6592ded735">

이 상태에요.

Reply:
```swift
 .map { $0.toDomain() }
```
이렇게 바꾸면 어떻게 되나요?

'map'이랑 여는 중괄호 사이에 공백 1칸이랑 
$0.toDomain() 양옆으로 공백 1칸이요!

Reply:
<img width="645" alt="스크린샷 2024-03-06 오후 5 30 03" src="https://github.com/wakmusic/wakmusic-iOS/assets/37323252/ae9cac61-630b-4ae0-914b-dd54b79965c4">

그러면 처음이랑 같아요.

Reply:
앗 'p'랑 '{' 이랑 사이에 공백 하나 넣어주세요!

Reply:
앗, 해결됐어요!